config/config_cmd.go

```diff
@@ -75,6 +99,155 @@ func redirectStdOut() (old, new *os.File) {
 	return
 }
 
+func printConfigTable(section Section, sectionName string, isSubsection bool, visitedSection map[string]bool, typeLink map[reflect.Type]string) {
```

Comment:
We need to add some unit tests for this...

Reply:
Maybe also do some refactoring to break down this function into smaller (easier to unit-test) ones?

Reply:
added it